### ISSUE TITLE
which Grafana dashboard to be used?

### ISSUE THREAD

**Issue by TungstenE2** [mxschmitt/fritzbox_exporter#35]
Hi all,

I installed the container and Prometheus is receining metrics from my 7490.

But when I install Grafana Dashboard there is no data shown.
https://grafana.com/grafana/dashboards/12579

Which Dashboard is working with this?

**Comment by TungstenE2** [comment]
push...

**Comment by DaZzLa09** [comment]
hi, did u find any fitting dashboard?

**Comment by TungstenE2** [comment]
no, so I stopped metrics for 7490. Let me know if you find one.

**Comment by pateball** [comment]
i think i'm late, but this one seems to work.
https://grafana.com/grafana/dashboards/11593-fritz-box/
